src/main/java/it/pagopa/interop/probing/eservice/registry/updater/model/Eservice.java

```diff
@@ -0,0 +1,145 @@
+* VERSION     DESCRIPTION OF CHANGE
+*-----------------------------------------------------------------------------
+** --/1.0  |  Initial Create.
+**---------|------------------------------------------------------------------
+***************************************************************************/
+
+package it.pagopa.interop.probing.eservice.registry.updater.model;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.UUID;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+
+import it.pagopa.interop.probing.eservice.registry.updater.util.EserviceState;
+import it.pagopa.interop.probing.eservice.registry.updater.util.EserviceTechnology;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * The Class Eservice.
+ */
+@Entity
+@Table(name = "eservices", uniqueConstraints = @UniqueConstraint(columnNames = { "eservice_id", "version_id" }))
+@TypeDef(name = "basePathType", typeClass = CustomStringArrayType.class)
+
+/**
+ * To string.
+ *
+ * @return the java.lang. string
+ */
+@Data
+
+/**
+ * Instantiates a new eservice.
+ */
+@NoArgsConstructor
+public class Eservice implements Serializable {
+
+	/** The Constant serialVersionUID. */
+	private static final long serialVersionUID = -2324588888685572113L;
+
+	/** The id. */
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "eservice_sequence")
+	@SequenceGenerator(name = "eservice_sequence", sequenceName = "eservice_sequence", allocationSize = 1)
+	@Column(updatable = false)
+	private Long id;
+
+	/** The base path. */
+	@NotNull
+	@Size(max = 255)
+	@Basic(optional = false)
+	@Column(name = "base_path", columnDefinition = "varchar(255) array")
+	@Type(type = "basePathType")
+	private String[] basePath;
+
+	/** The eservice name. */
+	@NotBlank
+	@Size(max = 255)
+	@Column(name = "eservice_name")
+	private String eserviceName;
+
+	/** The technology. */
+	@NotNull
+	@Column(name = "eservice_technology")
+	@Enumerated(EnumType.STRING)
+	private EserviceTechnology technology;
+
+	/** The eservice id. */
+	@NotNull
+	@Column(name = "eservice_id")
+	private UUID eserviceId;
+
+	/** The polling end time. */
+	@NotNull
+	@Column(name = "polling_end_time", columnDefinition = "TIMESTAMP with time zone")
+	private OffsetDateTime pollingEndTime = OffsetDateTime.of(OffsetDateTime.now().getYear(),
```

Comment:
[Blocking] shouldn't `polling_end_time` be just an `OffsetTime`?

Reply:
right, done

Reply:
this is still `OffsetDateTime`